parkings/api/enforcement/valid_parking.py

```diff
@@ -141,3 +149,26 @@ def get_grace_duration(default=datetime.timedelta(minutes=15)):
     value = getattr(settings, 'PARKKIHUBI_TIME_OLD_PARKINGS_VISIBLE', None)
     assert value is None or isinstance(value, datetime.timedelta)
     return value if value is not None else default
+
+
+class ValidParkingPermitViewSet(ValidParkingViewSet):
```

Comment:
Unifying the naming here might be good, see `ValidParkingAndPermitSerializer `, which suggests that the name for this class should be `ValidParkingAndPermitViewSet `.